client/package.json

```diff
@@ -33,10 +32,13 @@
     "@angular/cli": "~9.0.3",
     "@angular/compiler-cli": "~9.0.2",
     "@angular/language-service": "~9.0.2",
-    "@types/node": "^13.7.4",
+    "@nrwl/cypress": "^9.4.5",
```

Comment:
What are the `@nrwl` things and why do we need them? I would have thought we could have just done something like `npx cypress run` to run the Cypress tests?

Reply:
the `@nrwl/cypress` package is an Angular CLI builder/schematic and is what allows us to have Cypress run with an `ng` command that automatically builds the project and runs it while passing in its local URL as the base URL to Cypress. It also handles re-running the tests not only when the tests themselves change (like Cypress normally does) but also when the application code changes using `ng e2e --watch` (or `ng cypress --watch` if that's what you set your command name to.